### PR TITLE
Docs: updated README.md

### PR DESCRIPTION
**Pull request by storypku**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +43,11 @@ Apollo is loaded with new modules and features but needs to be calibrated and co
 
 * Ubuntu 18.04
 
-* NVIDIA driver version 440.33.01 and above ([Web link](https://www.nvidia.com/Download/index.aspx?lang=en-us)), and CUDA version 10.2
+* NVIDIA driver version 440.33.01 and above ([Web link](https://www.nvidia.com/Download/index.aspx?lang=en-us))
 
 * Docker-CE version 19.03 and above ([Official doc](https://docs.docker.com/engine/install/ubuntu/))
 
-* NVIDIA container toolkit ([Official doc](https://github.com/NVIDIA/nvidia-docker))
+* NVIDIA Container Toolkit ([Official doc](https://github.com/NVIDIA/nvidia-docker))
```
Comment (storypku) [inline README.md:51]: Where should we mention Apollo Software Installation Guide, here or the Installation section below ?

File: README.md [hunk 2]
```diff
@@ -85,52 +85,52 @@ Apollo Engineering team. Your safety is our #1 priority,
...
 
-![image alt text](docs/demo_guide/images/Hardware_connection_3_5_1.png)
+![](docs/demo_guide/images/Hardware_connection_3_5_1.png)
 
 * **Software Overview**
 
-![image alt text](docs/demo_guide/images/Apollo_3_5_software_architecture.png)
+![](docs/demo_guide/images/Apollo_3_5_software_architecture.png)
 
 ## Installation
```
Comment (storypku) [inline README.md:136]: Should we add Software Installation Guide here ?

File: README.md (hunk 1 above)
Comment (jinghaomiao) [reply README.md:51]: We don't need to introduce the software installation guide in prerequisites.

File: README.md (hunk 2 above)
Comment (jinghaomiao) [reply README.md:136]: Software installation guide is referenced in quickstart. You can add software installation guide in "installation" section  to replace step 2 and 3.

**Review by jinghaomiao — CHANGES_REQUESTED** [review]

**Review by jinghaomiao — APPROVED** [review]

**Pull request merged**